wal/src/table_kv_impl/table_unit.rs

```diff
@@ -407,11 +406,11 @@ impl TableUnit {
                 region_id,
                 table_id,
             )? {
-                last_sequence = seq;
+                return Ok(seq);
```

Comment:
Add a comment to describe why return here.

Reply:
There have been comments originally.